src/skin/popup.css

```diff
@@ -276,6 +272,12 @@ font-size: 16px;
     margin: 0;
     padding-top: 10px;
 }
+#pbInstructions :not(#options_domain_list_trackers) a {
```

Comment:
Options might instead use `#options_domain_list_one_tracker` or `#options_domain_list_no_trackers`, depending on the number of known domains.